src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -57,6 +63,12 @@
       <NgenArchitecture>All</NgenArchitecture>
       <NgenPriority>3</NgenPriority>
     </ProjectReference>
+    <ProjectReference Include="..\Razor\RazorVisualStudio.csproj">
+      <Project>{0c0eeb55-4b6d-4f2b-b0bb-b9eb2ba9e980}</Project>
+      <Name>RazorVisualStudio</Name>
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
+      <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
```

Comment:
I notice that the IN VS assemblies also seem to have:
```
<Ngen>true</Ngen>
        <NgenArchitecture>All</NgenArchitecture>		        <NgenArchitecture>All</NgenArchitecture>
        <NgenPriority>3</NgenPriority>		        <NgenPriority>3</NgenPriority>
```

Attached to them. I'm going to go ahead and provisionally add that to mine since its there for all the assemblies that live in VS.